src/cli.js

```diff
@@ -69,7 +70,7 @@ async function main() {
     await inherit()
     return
   }
-  const done = log.timedTask(kleur.bold().bgGreen(' lazyrepo '))
+  const done = log.timedTask(kleur.bold().bgGreen(` lazyrepo v${VERSION}`))
```

Comment:
@ds300 I'm not entirely sure about this format. Let me know if you have any suggestions on where we should include a version number.

Reply:
I'm not tied to this green thing with colons in any way, do you want to play around and come up with something different that works better with a version string? Feel free to experiment with color, boldness, backgrounds, whatever.

If that kinda thing is not your bag I'm happy to merge this and maybe tweak it later. Lemme know :)

Reply:
I'd be happy to experiment with it and provide a few variants. Will let you know when I have something :)